### PR TITLE
Add media keys to InputKey enum

### PR DESCRIPTION
**Pull request by Game4all**
- [x] Requires a new **osuTK** nuget release

Adds media keys to InputKey enum, allowing them to be used in keybindings.

### CODE REVIEW

File: osu.Framework/Input/Bindings/KeyCombination.cs [hunk 1]
```diff
@@ -227,6 +227,27 @@ private string getReadableKey(InputKey key)
...
+                    return "Vol. Up";
+
+                case InputKey.Stop:
+                    return "Media Stop";
+
+                case InputKey.PlayPause:
+                    return "Media Play";
+
+                case InputKey.TrackNext:
+                    return "Media Previous";
```
Comment (UselessToucan) [inline osu.Framework/Input/Bindings/KeyCombination.cs]:
```suggestion
                    return "Media Next";
```

File: osu.Framework/Input/Bindings/KeyCombination.cs [hunk 2]
```diff
@@ -227,6 +227,27 @@ private string getReadableKey(InputKey key)
...
+
+                case InputKey.TrackPrevious:
+                    return "Media Next";
```
Comment (UselessToucan) [inline osu.Framework/Input/Bindings/KeyCombination.cs]:
```suggestion
                    return "Media Previous";
```

Comment (Game4all) [reply osu.Framework/Input/Bindings/KeyCombination.cs]: whoops 😅

**Comment by peppy** [comment]
I've just done a nuget build.

**Comment by Game4all** [comment]
Just realized this will probably mess up some databased keybinds on osu!-side as media keys have been inserted in the middle of the key enum

**Comment by peppy** [comment]
Oh, you actually changed the explicit ints.

THis cannot happen.

**Comment by Game4all** [comment]
As I understand, OpenTK enum values are 1:1 mapped to InputKey enum values.

Even if I move the media keys at the end of the enum of the OpenTK Key enum (just before LastKey), adjusting explicit InputKey enum will still be necessary.

 It will probably reduce the number of broken bindings drastically but there isn't a way to assure no bindings will be broken AFAIK.

**Comment by peppy** [comment]
Okay so one of the two needs to happen:

- Rollback osutk changes and add keys at end of enum there
- remap the osutk range to correct inputkey range (without inserted new items).

**Comment by Game4all** [comment]
Let's go for the first option

**Comment by Game4all** [comment]
Is there any reason why `LastKey` is included in `InputKey` enum, as this enum value isn't really intended to be user-used ? Guess I could get it off the enum while i'm messing with it if there is no specific reason.

**Comment by peppy** [comment]
It's just a marker value. I'm not sure it's required but no harm in leaving it there.

Looks like some tests broke as a result of this change, please check on them.

**Pull request merged**